packages/http/src/validator/validators/utils.ts

```diff
@@ -89,6 +89,32 @@ export const convertAjvErrors = (errors: NonEmptyArray<ErrorObject>, severity: D
     })
   );
 
+const validationsFunctionsCache = new WeakMap<JSONSchema, WeakMap<object, ValidateFunction>>();
+const EMPTY_BUNDLE = { _emptyBundle: true };
```

Comment:
do we really need this var? We use it only in one place. 
If you prefer to keep it, can you at least move it inside `getValidationFunction`.

Reply:
`WeakMap` accepts objects as a key only. In case bundle is empty or null we use `EMPTY_BUNDLE` as a key in weak map. We need to have it defined once since `WeakMap` cached by reference not object equality.

Reply:
> we need to have it defined only once
that's what I'm talking about do we need to define this var? 

```
const bundleKey = typeof bundle === 'object' && bundle !== null ? bundle : { _emptyBundle: true };
```

Reply:
<img width="400" alt="Zrzut ekranu 2021-09-21 o 14 11 03" src="https://user-images.githubusercontent.com/88896271/134168061-947885c2-1cd9-4491-afc0-c5e6f42415d4.png">

So your proposal would not work.

Reply:
ah I see now, gotcha. cool make sense.